.circleci/config.templ.yml

```diff
@@ -80,7 +80,7 @@ commands:
     description: "Install riot"
     steps:
       # Make sure we install and run riot on Python 3
-      - run: pip3 install riot==0.19.1
+      - run: pip3 install virtualenv==20.26.6 riot==0.19.1
```

Comment:
nit: Can we add a comment here so we know why we're pinning this (and so we can clean it up/update it when we remove support for 3.7)?